src/app/(nextjs_migration)/(authenticated)/user/layout.tsx

```diff
@@ -27,6 +29,32 @@ const MainLayout = async (props: Props) => {
     return <SignInButton autoSignIn />;
   }
 
+  const accountId = session?.user?.subscriber?.fxa_uid;
+
+  let userId;
+  if (accountId) {
+    // If the user is logged in, use a UUID based on the user's subscriber ID.
+    // Note: we may want to use the FxA UID here, but we need approval for that first.
+    // No need to store in a cookie - this is derived from subscriber ID.
+    if (!process.env.NIMBUS_UUID_NAMESPACE) {
```

Comment:
Below (L68) we used `NEXT_PUBLIC_*` prefix, does this need to be `NEXT_PUBLIC_NIMBUS_UUID_NAMESPACE`?
We also seem to be using `process.env.SERVER_URL` but not `process.env.NEXT_PUBLIC_SERVER_URL` so maybe not. 🤷 

https://github.com/mozilla/blurts-server/blob/bb08ba7f28649e0ff2dfc4e57491409ad0ca4e01/src/app/(nextjs_migration)/(authenticated)/user/layout.tsx#L68